docs/getting-started/release-notes/v2_10_0.md

```diff
@@ -47,20 +36,26 @@ Zowe Version 2.10.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
+- Fixed the `zowe files create data-set` command failing when no additional options are specified.
+- Added check for invalid block size when creating a sequential data set. ([#1439](https://github.com/zowe/zowe-cli/issues/1439))
+- Added the ability to list all data set members when some members have invalid names.
+- Removed extra calls to list data sets matching patterns if authentication to z/OSMF fails.
 
-#### DB2 Plug-in for Zowe CLI
-
-#### MQ Plug-in for Zowe CLI
+#### Zowe CLI Imperative Framework
 
-#### IMS Plug-in for Zowe CLI
+- Enabled NextVerFeatures.useV3ErrFormat() to form the right environment variable name even if Imperative.init() has not been called.
 
-### Zowe Explorer
+- Handle logic for if a null command handler is provided
 
 ### Vulnerabilities fixed
 
 Zowe discloses fixed vulnerabilities in a timely manner giving you sufficient time to plan your upgrades. Zowe does not disclose the vulnerabilities fixed in the latest release as we respect the need for at least 45 days to decide when and how you upgrade Zowe. When a new release is published, Zowe publishes the vulnerabilities fixed in the previous release. For more information about the Zowe security policy, see the [Security page](https://www.zowe.org/security.html) on the Zowe website.
 
 The following security issues were fixed by the Zowe security group in version 2.9.
+
+- BDSA-2023-1491
+- CVE-2023-33546
+- CVE-2022-1471 (BDSA-2022-3447)
+- BDSA-2023-0953
+- CVE-2023-20883 (BDSA-2023-1225)
+- CVE-2023-20873
```

Comment:
We may want someone the @zowe/api-mediation to confirm this block before merging. 😋

Reply:
It's in the security file, so it should be good. Either way, the release notes will be shared for squad lead review so I'm hoping someone checks it then.